docs/output_fields.rst

```diff
@@ -435,3 +435,81 @@ Borehole samples (grondmonsters)
     fractie,10,float,0
     methode,10,string,ZEEFPROEF
 
+Bodemsites
+--------------------------------
+  .. csv-table:: Bodemsites
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemsite,1,string,https://www.dov.vlaanderen.be/data/bodemsite/2013-000180
+	naam,1,string,Meise_Neerpoorten
+	datum,1,date,nan
+	beschrijving,1,string,grasland
+	waarnemingsdatum,10,date,nan
+	invoerdatum,10,date,nan
+
+
+BodemLocaties
+--------------------------------
+  .. csv-table:: Bodemlocaties
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemlocatie,1,string,https://www.dov.vlaanderen.be/data/bodemlocatie/2011-000002
+	naam,1,string,STARC_4
+	type,1,string,profielput
+	datum,1,date,nan
+	doel,1,string,archeologische landschappelijke profielputten
+	x,1,float,206553.85
+	y,1,float,168891.11
+	z,1,float,44.00
+	erfgoed,1,boolean,true
+	bodemstreek,1,string,Zandleemstreek
+	waarnemingsdatum,10,date,nan
+	invoerdatum,10,date,nan
+	educatieve_waarde,10,string,ZEER
+
+
+Bodemmonsters
+--------------------------------
+  .. csv-table:: Bodemmonsters
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemmonster,1,string,https://www.dov.vlaanderen.be/data/bodemmonster/1964-264869
+	identificatie,1,string,KART_PROF_073E/12_H2_M1
+	datum_monstername,1,date,1964-11-12
+	tijdstip_monstername,10,time,NaN
+	type,1,string,ENK
+	monsternamedoor,1,string,Centrum voor Grondonderzoek (C.V.G.)
+	techniek,1,string,NaN
+	condities,1,string,Zie scan analoge profielbeschrijving
+	van,1,float,30
+	tot,1,float,45
+	labo,1,string,Centrum voor Grondonderzoek (C.V.G.)
+
+
+Bodemobservaties
+--------------------------------
+  .. csv-table:: Bodemobservaties
+    :header-rows: 1
+
+    Field,Cost,Datatype,Example
+	pkey_bodemobservatie,1,string,https://www.dov.vlaanderen.be/data/bodemobservatie/2019-349078
+	pkey_bodemlocatie,1,string,https://www.dov.vlaanderen.be/data/bodemlocatie/1952-007078
+	Aan,1,string,bodemlocatie/1952-007078
+	pkey_parent,1,string,https://www.dov.vlaanderen.be/data/bodemlocatie/1952-007078
```

Comment:
Most of the time, the user will use the bodemlocatie as the primary object to search on. Therefore, the bodemmonster and bodemobservatie haves the pkey_bodemlocatie.
If it will be not available, the user needs to go (possibly) from bodemobservatie to parent bodemmonster to parent bodemdiepteinterval to parent bodemlocatie in order to get the information of the bodemlocatie.

The reason to have the pkey_parent is to know what's the parent of the bodemobservatie/bodemmonster.
For bodemobservatie, the parent can be: bodemsite, bodemlocatie, bodemdiepteinterval of bodemmonster
For bodemmonster, the parent can be: bodemsite, bodemlocatie, bodemdiepteinterval.

Reply:
Ok, that makes sense. It would (possibly) require a lot of extra queries to get to the Bodemlocatie of an object, so we might benefit from adding the direct link?

In that case I would keep `pkey_bodemlocatie` and `pkey_parent`. The `Aan` seems not very interesting to me (duplicate with pkey_parent)?